styles/mixins/dark-mode.scss

```diff
@@ -0,0 +1,38 @@
+@mixin dark-mode($global: false) {
```

Comment:
How does this work when we want to override the theme for the custom component editor?

Reply:
Like via the iconColor prop? If so… the iconColor prop adds `--Card-icon-color: {{color}}` to the inline style tag, which would overwrite anything in the css file.

Lmk if I'm not understanding your questions tho (I might not be…)!!

Reply:
https://github.com/readmeio/readme/blob/next/packages/react/src/routes/Dash/CustomBlocks/Form/ComponentEditor/style.module.scss#L78